README.md

```diff
@@ -32,154 +38,209 @@ Rouge::Themes::Base16.mode(:light).render(scope: '.highlight')
+  the given formatter with div wrappers generally expected by stylesheets
+  designed for Pygments.
+
+* `Rouge::Formatters::HTMLTable.new(formatter, opts={})` will output an HTML
+  table containing numbered lines similar to `Rouge::Formatters::HTMLLineTable`,
+  except that the table from this formatter has just a single table-row.
+  Therefore, while the table is more DOM-friendly for JavaScript scripting, long
+  code lines will mess with the column alignment. Options are:
     * `start_line: 1` - the number of the first line
     * `line_format: '%i'` - a `sprintf` template for the line number itself
     * `table_class: 'rouge-table'` - a CSS class for the table
     * `gutter_class: 'rouge-gutter'` - a CSS class for the gutter
     * `code_class: 'rouge-code'` - a CSS class for the code column
-* `Rouge::Formatters::HTMLLegacy.new(opts={})` is a backwards-compatibility class intended
-  for users of rouge 1.x, with options that were supported then. Options are:
+
+* `Rouge::Formatters::HTMLLegacy.new(opts={})` is a backwards-compatibility
+  class intended for users of Rouge 1.x, with options that were supported then.
+  Options are:
     * `inline_theme: nil` - use an HTMLInline formatter with the given theme
     * `line_numbers: false` - use an HTMLTable formatter
     * `wrap: true` - use an HTMLPygments wrapper
-    * `css_class: 'codehilite'` - a CSS class to use for the pygments wrapper
-* `Rouge::Formatters::Terminal256.new(theme)`
-  * `theme` must be an instnce of `Rouge::Theme`, or a `Hash` structure with `:theme` entry
+    * `css_class: 'codehilite'` - a CSS class to use for the Pygments wrapper
 
-#### Lexer options
-##### debug: false
-Print a trace of the lex on stdout
+* `Rouge::Formatters::Terminal256.new(theme)` is a formatter for generating
+  highlighted text for use in the terminal. `theme` must be an instance of
+  `Rouge::Theme`, or a `Hash` structure with `:theme` entry.
 
-##### parent: ''
-Allows you to specify which language the template is inside
+### Lexer Options
 
-#### CSS theme options
-##### scope: '.highlight'
-CSS selector that styles are applied to, e.g. `Rouge::Themes::MonokaiSublime.render(scope: 'code')`
+* `debug: false` will print a trace of the lex on stdout.
 
-Rouge aims to be simple to extend, and to be a drop-in replacement for pygments, with the same quality of output. Also, Rouge ships with a `rougify` command which allows you to easily highlight files in your terminal:
+* `parent: ''` allows you to specify which language the template is inside.
 
-``` bash
-$ rougify foo.rb
-$ rougify style monokai.sublime > syntax.css
-```
+### CSS Options
 
-### Advantages to pygments.rb
-* No need to [spawn Python processes](https://github.com/tmm1/pygments.rb).
-* We're faster in [almost every measure](https://github.com/rouge-ruby/rouge/pull/41#issuecomment-223751572)
+* `scope: '.highlight'` sets the CSS selector to which styles are applied, eg.
```

Comment:
Very nitpicky but it should be "e.g." instead of "eg.".

Reply:
This is my bad. I'm not sure how I got into this habit. I thought perhaps it might have been an Australianism but after some quick research almost every resource I can find (for all English variations) recommends 'e.g.'. Will update :)